input/definitions/Encounter/CodeSystem-encounter-link-type.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<CodeSystem xmlns="http://hl7.org/fhir">
+  <id value="encounter-link-type"/>
+  <meta>
+    <lastUpdated value="2025-11-06T00:00:00+00:00"/>
+    <profile value="http://hl7.org/fhir/StructureDefinition/shareablecodesystem"/>
+  </meta>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-wg">
+    <valueCode value="pa"/>
+  </extension>
+  <extension url="http://hl7.org/fhir/StructureDefinition/structuredefinition-standards-status">
+    <valueCode value="draft"/>
```

Comment:
If the extension is trial-use, I think the code system should be too?

Reply:
It's only an example binding to provide an indicative set that could be used.
And as a first pass on the values, I think draft makes sense.